Source/Http/RequestParameter.swift

```diff
@@ -21,30 +21,36 @@
 import Foundation
 
 struct RequestParameter {
-    private var storage: [String: AnyObject] = [:]
+    private var storage: [String: Any] = [:]
     
-    init(_ parameters: [String: Any?] = [:]) {
+    init(_ parameters: [String: Any] = [:]) {
```

Comment:
Please note this class is internal, and just a utility for strip the default nil API parameter. It won't impact the usage of request API.
My concern is code after change is not straightforward, and hard to maintain.